cvat/settings/production.py

```diff
@@ -12,3 +12,4 @@
 # https://github.com/moggers87/django-sendfile2
 SENDFILE_BACKEND = 'django_sendfile.backends.nginx'
 SENDFILE_URL = '/'
+CORS_ALLOW_ALL_ORIGINS = True
```

Comment:
Just for your information: https://github.com/adamchainz/django-cors-headers/blob/6f3fe99724bf42e57b062c8959792ca94c669ab0/README.rst#cors_allow_all_origins-bool

Setting this to True can be dangerous, as it allows any website to make cross-origin requests to yours. Generally you'll want to restrict the list of allowed origins with CORS_ALLOWED_ORIGINS or CORS_ALLOWED_ORIGIN_REGEXES.

It is a bad security practice. CVAT repo doesn't need the change, you don't need as well. It is better to list some specific origins. For the development purpose, it is better to run your own CVAT instance and specify additional settings.

@azhavoro , FYI.